akka-docs/src/test/scala/docs/stream/operators/SourceFromFuture.scala

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.{Done, NotUsed}
+import akka.stream.scaladsl._
+
+import scala.concurrent.Future
+
+object SourceFromFuture {
```

Comment:
Perhaps it would be OK to combine all examples for `Source` operators in one `Source.scala`? This is OK too, though!

Reply:
Sure I can update the PR with these changes.